modules/installation-nutanix-config-yaml.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/configuring-iam-nutanix.adoc
+
+:_content-type: REFERENCE
+[id="installation-nutanix-config-yaml_{context}"]
+= Sample customized install-config.yaml file for Nutanix
+
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+compute: <2>
+- hyperthreading: Enabled <3>
+  name: worker
+  replicas: 3
+  platform:
+    nutanix: <4>
+      cpus: 2
+      coresPerSocket: 2
+      memoryMiB: 8196
+      osDisk:
+        diskSizeMiB: 120
```

Comment:
Nutanix is using disSizeMiB, 120 is too small in that case,  maybe we need to change to 120000. I also opened a thread why Nutanix using disSizeMiB rather than disSizeGB here https://coreos.slack.com/archives/C0211848DBN/p1654844404353659.

Reply:
https://github.com/openshift/installer/pull/5998 has now been merged; we can switch this to GiB

Reply:
Thank you. Done.